tests/unittests/CoreText/CTFontTests.mm

```diff
@@ -147,6 +147,7 @@ virtual void TearDown() {
     EXPECT_NE(0, CTFontGetUnderlineThickness(font));
     EXPECT_NE(0, CTFontGetCapHeight(font));
     EXPECT_NE(0, CTFontGetXHeight(font));
+    EXPECT_EQ(0, CTFontGetLeading(font));
```

Comment:
discussed offline, holding off til CGFont

---
In reply to: [80785199](https://github.com/Microsoft/WinObjC/pull/1039#discussion_r80785199) [](ancestors = 80785199)